src/Factory/PdoAdapterFactory.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Laminas\ApiTools\OAuth2\Factory;
 
 use Interop\Container\ContainerInterface;
```

Comment:
These reported CS issues need to be suppressed: I think we may want to run `phpcs -s` to see **which** rule it is, that is raising them.

Reply:
Once the build is green due to this, I'm happy to merge+release :)

Reply:
For now, let's just drop it from our local phpcs config (this repo) via exclusion. Should be like https://github.com/Ocramius/ProxyManager/blob/8846623bea8749ded206db3ce701e4e508d516db/phpcs.xml.dist#L20-L21